src/ngx_http_lua_regex.c

```diff
@@ -214,8 +383,149 @@ ngx_http_lua_ffi_set_jit_stack_size(int size, u_char *errstr,
                    - errstr;
     return NGX_ERROR;
 
+#endif
+}
+
+
+#if (NGX_PCRE2)
+static void
+ngx_http_lua_regex_jit_compile(ngx_http_lua_regex_t *re, int flags,
+    ngx_pool_t *pool, ngx_http_lua_main_conf_t *lmcf,
+    ngx_http_lua_regex_compile_t *re_comp)
+{
+    ngx_int_t    ret;
+    ngx_pool_t  *old_pool;
+
+    if (flags & NGX_LUA_RE_MODE_JIT) {
+        old_pool = ngx_http_lua_pcre_malloc_init(pool);
+        ret = pcre2_jit_compile(re_comp->regex, PCRE2_JIT_COMPLETE);
+
+        if (ret != 0) {
+            ngx_log_error(NGX_LOG_INFO, ngx_cycle->log, 0,
+                          "pcre2_jit_compile() failed: %d in \"%V\", "
+                          "ignored",
+                          ret, &re_comp->pattern);
+
+#if (NGX_DEBUG)
+
+        } else {
+            /* pcre2 jit compiled successfully */
+            ngx_log_debug0(NGX_LOG_DEBUG_HTTP, ngx_cycle->log, 0,
```

Comment:
@swananan Here, I'd change the message

Appologies it seems I can't link to this line directly.

Reply:
> @swananan Here, I'd change the message
> 
> Appologies it seems I can't link to this line directly.

Oh, I understand now. I kept this log solely for passing the old pcre jit related test cases. I will correct this log and make the necessary modifications to the related test cases. Thanks for your comment!